### PR TITLE
Add Navigating the Menu subsection

### PR DESCRIPTION
**Pull request by chrisib**
Make a separate section with menu navigation instructions so they're easier for beginners to find.

### CODE REVIEW

File: software/programming_instructions.md [hunk 1]
```diff
@@ -117,12 +117,21 @@ Now you have access to the inputs and outputs using easy methods, which you can
...
+
 One of the scripts that is installed with the menu system is named '~ Calibrate', and it requires you to send precise voltages to the module to calibrate it for the future, allowing you to input and output precise values from your module.  
 This is entirely optional and it will work with a usable degree of accuracy without calibration, however if you do want to then move to the ['Calibrate the module'](#calibrate-the-module) step.
 
 
+#### Navigating the menu
+
+To navigate the menu use the right knob. Turning clockwise will scroll down and turning anticlockwise will scroll up.
+
+To run the selected program, press the left button once.  The last-run program will automatically start the next time you power-on your EuroPi.
```
Comment (mjaskula) [inline software/programming_instructions.md:130]: [required] Either button works to launch a script.

**Review by mjaskula — CHANGES_REQUESTED** [review]
Hey, @chrisib thank you for these document changes. It's really great when someone can identify and fix a problem that they had. Open source is great.

I've identified a small change that we should make before merging, but other than that it looks good.

Thanks again.

File: software/programming_instructions.md (hunk 1 above)
Comment (chrisib) [reply software/programming_instructions.md]: I just pushed a fix

**Review by mjaskula — APPROVED** [review]
Thanks for the change!

**Pull request merged**